dts/bindings/misc/nxp,imx-rdc.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NXP i.MX Resource Domain Controller (RDC) node.
+  The RDC provides robust support for the isolation of destination
+  memory mapped locations such as peripherals and memory to a single
+  core, a bus master, or set of cores and bus masters.
+
+compatible: "nxp,imx-rdc"
```

Comment:
Can we drop the family name from this compatible? While the RDC peripheral is probably only being used on iMX family devices, in general I want to avoid this kind of family naming on IP blocks

Reply:
make sense, will update. Thanks!